lms/djangoapps/discussion_api/forms.py

```diff
@@ -2,12 +2,38 @@
 Discussion API forms
 """
 from django.core.exceptions import ValidationError
-from django.forms import BooleanField, CharField, Form, IntegerField, NullBooleanField
+from django.forms import (
+    BooleanField,
+    CharField,
+    Field,
+    Form,
+    IntegerField,
+    MultipleHiddenInput,
+    NullBooleanField,
+)
 
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.locator import CourseLocator
 
 
+class TopicIdField(Field):
+    """
+    Field for a list of topic_ids
+    """
+    def __init__(self, *args, **kwargs):
+        self.field_type = CharField()
+        self.widget = MultipleHiddenInput
```

Comment:
It looks like the built-in fields have this as a class attribute. Is there a reason you didn't do that?